detectable/src/main/java/com/blackduck/integration/detectable/detectables/opam/buildexe/OpamBuildDetectable.java

```diff
@@ -0,0 +1,64 @@
+package com.blackduck.integration.detectable.detectables.opam.buildexe;
+
+import com.blackduck.integration.common.util.finder.FileFinder;
+import com.blackduck.integration.detectable.Detectable;
+import com.blackduck.integration.detectable.DetectableEnvironment;
+import com.blackduck.integration.detectable.ExecutableTarget;
+import com.blackduck.integration.detectable.detectable.DetectableAccuracyType;
+import com.blackduck.integration.detectable.detectable.Requirements;
+import com.blackduck.integration.detectable.detectable.annotation.DetectableInfo;
+import com.blackduck.integration.detectable.detectable.exception.DetectableException;
+import com.blackduck.integration.detectable.detectable.executable.resolver.OpamResolver;
+import com.blackduck.integration.detectable.detectable.result.DetectableResult;
+import com.blackduck.integration.detectable.detectable.result.ExecutableNotFoundDetectableResult;
+import com.blackduck.integration.detectable.detectable.result.PassedDetectableResult;
+import com.blackduck.integration.detectable.extraction.Extraction;
+import com.blackduck.integration.detectable.extraction.ExtractionEnvironment;
+import com.blackduck.integration.executable.ExecutableRunnerException;
+
+import java.io.File;
+import java.util.List;
+
+
+@DetectableInfo(name = "Opam CLI", language = "OCaml", forge = "opam", accuracy = DetectableAccuracyType.HIGH, requirementsMarkdown = "A .opam file and the opam executable.")
```

Comment:
For `requirementsMarkdown` we tend to have a special format which in this case would look like `File: an opam file with .opam extension. Executable: opam.`.